sphinx/writers/latex.py

```diff
@@ -841,10 +841,15 @@ def depart_rubric(self, node: Element) -> None:
     def visit_footnote(self, node: Element) -> None:
         self.in_footnote += 1
         label = cast(nodes.label, node[0])
+        if 'auto' not in node.attributes:
+            self.body.append('\\sphinxstepexplicit ')
         if self.in_parsed_literal:
             self.body.append('\\begin{footnote}[%s]' % label.astext())
         else:
             self.body.append('%%\n\\begin{footnote}[%s]' % label.astext())
+        if 'auto' not in node.attributes:
+            self.body.append('\\phantomsection'
+                             '\\label{\\thesphinxscope.%s}%%\n' % label.astext())
```

Comment:
In this function it seems on next line the addition of `\sphinxAtStartFootnote` did not really pay attention to whether in parsed_literal or not. Apparently in parsed_literal we want to avoid adding a `\n`. Maybe we should check that.